### PR TITLE
perf: add meta tags

### LINKED ISSUE #418: feat: Use meta tags to control how your website appears in search results
**Is your feature request related to a problem? Please describe.**
Feature

**Describe the solution you'd like**
I want to add og meta tags similar to vitess website to improve how website looks when shared

**Additional context**

P.S - I want to work on it.

### PR DESCRIPTION
**Pull request by Lucif3r-in**
Added meta tags to improve website visibility when shared.
We can also add the tags to the react-version of the website if approved.
Closes #418

### CODE REVIEW

**Review by frouioui — CHANGES_REQUESTED** [review]
Thanks @Lucif3r-in for this first contribution. The web server under `./go/server` will soon be removed. It has been replaced by the React project in `./website`. Can you add the meta tags there?

**Comment by Lucif3r-in** [comment]
> Thanks @Lucif3r-in for this first contribution. The web server under `./go/server` will soon be removed. It has been replaced by the React project in `./website`. Can you add the meta tags there?

Sure I was going to do that in the next step, was just waiting for your approval. Will do it ASAP.

**Comment by Lucif3r-in** [comment]
@frouioui @GuptaManan100 Done changes as suggested.

File: website/public/img/favicon.png
Comment (frouioui) [inline website/public/img/favicon.png:1]: Let's move both images to the `website/src/assets` directory, we already have a few images there. At the same time, you can remove the `website/public/` directory entirely to avoid future confusion.

**Review by frouioui — APPROVED** [review]
Approving this PR for now, I'll merge it once all comments have been addressed. Thanks @Lucif3r-in for this helpful contribution!

File: website/public/img/favicon.png
Comment (Lucif3r-in) [reply website/public/img/favicon.png:1]: Ok, on it!!

**Comment by Lucif3r-in** [comment]
> Approving this PR for now, I'll merge it once all comments have been addressed. Thanks @Lucif3r-in for this helpful contribution!

Sure, I have made the suggested changes🚀🚀

**Review by frouioui — APPROVED** [review]

**Comment by frouioui** [comment]
awesome, thank you @Lucif3r-in !

**Pull request merged**
